src/modules/Map/Map/components/SearchControl/SearchInput.js

```diff
@@ -8,9 +8,11 @@ export const SearchInput = ({
   onKeyPress,
   onFocus,
   loading,
+  placeholder,
 }) => (
   <Card className="search-input">
     <InputGroup
+      placeholder={placeholder}
```

Comment:
What about:

```suggestion
  ...rest
}) => (
  <Card className="search-input">
    <InputGroup
      {...rest}
```

or maybe something like:

```js
export const SearchInput = ({
  query: value,
  onClose,
  loading,
  cardProps = {},
  ...props
}) => (
  <Card className="search-input" {...cardProps}>
    <InputGroup
      rightElement={(
        <Button
          icon="small-cross"
          minimal
          loading={loading}
          onClick={onClose}
        />
      )}
      {...props}
    />
  </Card>
);
```